Ledger application migration (#3681)
w3f/polkadot-wiki: docs/general/ledger.md

- Ledger Live is installed and at version 2.1 or newer (see settings -> about to find out if you're
  up to date).
- A Chromium-based web browser is installed that you can use to access [Polkadot-JS Apps][].

## Using Ledger Live

Check [this support article](https://support.polkadot.network/support/solutions/articles/65000175822-how-to-use-polkadot-and-stake-with-ledger-live) to learn how to use Polkadot with ledger live.

:::info

Ledger Live will only show the main account with BIP44 path 44'/354'/0'/0'/0'. This means that if you create a [derived account using Polkadot JS](#using-on-polkadot-js-apps) with a derivation path 44'/354'/0'/0'/1', it will not be displayed on the Ledger Live App. As a consequence it is not possible to transact with derived accounts using the Ledger Live App, but it is possible to do so using Polkadot JS. For more information about derived accounts and derivation paths check [the accounts page](../learn/learn-accounts.md).

:::

## Using on Polkadot-JS Apps

### Loading Your Account

> :::info Ledger Live should be off while using Ledger with Polkadot-JS Apps as it can interfere with normal operation.

Reviewer: ```suggestion
:::info Ledger Live should be off while using Ledger with Polkadot-JS UI as it can interfere with normal operation.
```

:::

> You can import your Ledger account to [Polkadot Extension](https://polkadot.js.org/extension/) or to the [Polkadot JS UI](https://polkadot.js.org/apps/#/explorer). For instructions

Reviewer: ```suggestion
You can import your Ledger account to [Polkadot Extension](https://polkadot.js.org/extension/) or to the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer). For instructions
```

> on how to import Ledger accounts to the Polkadot JS extension read through [this support article](https://support.polkadot.network/support/solutions/articles/65000175387-how-to-add-your-ledger-through-the-polkadot-extension), while if you want to import Ledger accounts to the Polkadot JS UI you can consult [this other article](https://support.polkadot.network/support/solutions/articles/65000170812-how-to-add-ledger-account-through-the-polkadot-js-ui).

Reviewer: ```suggestion
on how to import Ledger accounts to the Polkadot Extension read through [this support article](https://support.polkadot.network/support/solutions/articles/65000175387-how-to-add-your-ledger-through-the-polkadot-extension), while if you want to import Ledger accounts to the Polkadot JS UI you can consult [this other article](https://support.polkadot.network/support/solutions/articles/65000170812-how-to-add-ledger-account-through-the-polkadot-js-ui).
```

#### Derivation paths

> When adding a Ledger account using the extension or the UI, you will be asked to select an `account type` and an `account index`. The first lets you select an account, while the second lets you pick a derivation path from that account - think of it like a formula from which child accounts are generated. When you are creating a Polkadot ledger account for the first time on Ledger Live with name `Polkadot 1`, this can be added to Polkadot JS using the 0/0 derivation path (i.e. account type = 0 and account index = 0). If then you add a second account called `Polkadot 2`, this will correspond to the 1/0 derivation path, and so on. We thus have multiple parent accounts that cam be viewed and used in both Ledger Live and Polkadot JS. Additionally, we can use Polkadot JS to created multiple children accounts from each parent account. For example, `Polkadot 1` with 0/0 derivation path can have child 0/1, 0/2, etc. that can be used within the UI. However, such children accounts cannot be used in Ledger Live, as it only scans through the parent accounts. So, keep in mind that on Ledger Live balances on the children accounts cannot be viewed, and you cannot use such accounts.

Reviewer: Fixed typo `cam be viewed` to `can be viewed`

Rephrased the last sentence

```suggestion
When adding a Ledger account using the extension or the UI, you will be asked to select an `account type` and an `account index`. The first lets you select an account, while the second lets you pick a derivation path from that account - think of it like a formula from which child accounts are generated. When you are creating a Polkadot ledger account for the first time on Ledger Live with name `Polkadot 1`, this can be added to Polkadot JS using the 0/0 derivation path (i.e. account type = 0 and account index = 0). If then you add a second account called `Polkadot 2`, this will correspond to the 1/0 derivation path, and so on. We thus have multiple parent accounts that can be viewed and used in both Ledger Live and Polkadot JS. Additionally, we can use Polkadot-JS UI to created multiple children accounts from each parent account. For example, `Polkadot 1` with 0/0 derivation path can have child 0/1, 0/2, etc. that can be used within the UI. However, such children accounts cannot be used in Ledger Live, as it only scans through the parent accounts. So, keep in mind that the balances on the children accounts cannot be viewed, and you will not be able to transact with those accounts on Ledger Live.
```

#### Connecting your ledger device

While using a ledger device to sign transactions, depending on your browser and its security settings, you might need
to confirm the USB connection through a popup like the one below:

![Display the device connection popup](../assets/ledger/query-device.png)

> If you are adding your Ledger nano for the first time click on the "Unknown device" line and the "Connect" button will become available. 

Reviewer: ```suggestion
If you are adding your Ledger Nano for the first time, click on the "Unknown device" line and the "Connect" button will become available. 
```

:::info Signature error message

> If you already connected your device but an error message appears before signing a transaction make sure that on your Ledger nano you have opened the Polkadot application. Visit [this support page](https://support.polkadot.network/support/solutions/articles/65000181994) for more information about signing transactions using your ledger.

Reviewer: ```suggestion
If you already connected your device but an error message appears before signing a transaction, make sure you have opened the Polkadot application on your Ledger Nano device. Visit [this support page](https://support.polkadot.network/support/solutions/articles/65000181994) for more information about signing transactions using your ledger.
```

:::

### Confirming the Address on your Device

> To display your Polkadot ledger account address on your Ledger nano you can follow the guidelines on [this support article](https://support.polkadot.network/support/solutions/articles/65000181854-how-to-confirm-your-account-address-on-your-ledger-device). Here you can scroll through and make sure the address matches to what is displayed on Polkadot-JS

Reviewer: ```suggestion
To display your Polkadot ledger account address on your Ledger Nano you can follow the guidelines on [this support article](https://support.polkadot.network/support/solutions/articles/65000181854-how-to-confirm-your-account-address-on-your-ledger-device). Here you can scroll through and make sure the address matches to what is displayed on Polkadot-JS UI
```
